### PR TITLE
[PREVIEW] RPE-733 Added zip filename to Queue message model

### PR DESCRIPTION
**Pull request by aliveni**
**Before creating a pull request make sure that:**

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RPE-733
https://tools.hmcts.net/jira/browse/RPE-655

### Change description ###
Updated Queue Message model to add zipFileName property

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/EnvelopeParserTest.java [hunk 1]
```diff
@@ -55,7 +57,7 @@ public void should_throw_an_exception_if_json_is_not_a_valid_envelope() throws E
     }
 
     @Test
-    public void should_throw_an_exception_if_invalid_json_is_passed() throws Exception {
+    public void should_throw_an_exception_if_invalid_json_is_passed() {
```
Comment (rkondratowicz) [inline src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/EnvelopeParserTest.java]: unrelated?

Comment (aliveni) [reply src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/EnvelopeParserTest.java]: yes, it is unrelated to my PR. just cleaned up as the exception is not being thrown
